pallets/identity/src/lib.rs

```diff
@@ -642,54 +773,60 @@ decl_module! {
         ///
         /// # Errors
         ///
-        #[weight = <T as Config>::WeightInfo::freeze_secondary_keys()]
-        pub fn freeze_secondary_keys(origin) -> DispatchResult {
+        #[pallet::weight(<T as Config>::WeightInfo::freeze_secondary_keys())]
+        #[pallet::call_index(8)]
+        pub fn freeze_secondary_keys(origin: OriginFor<T>) -> DispatchResult {
             Self::set_frozen_secondary_key_flags(origin, true)
         }
 
         /// Re-enables all secondary keys of the caller's identity.
-        #[weight = <T as Config>::WeightInfo::unfreeze_secondary_keys()]
-        pub fn unfreeze_secondary_keys(origin) -> DispatchResult {
+        #[pallet::weight(<T as Config>::WeightInfo::unfreeze_secondary_keys())]
+        #[pallet::call_index(9)]
+        pub fn unfreeze_secondary_keys(origin: OriginFor<T>) -> DispatchResult {
             Self::set_frozen_secondary_key_flags(origin, false)
         }
 
         // Manage generic authorizations
         /// Adds an authorization.
-        #[weight = <T as Config>::WeightInfo::add_authorization_full::<T::AccountId>(&data)]
+        #[pallet::weight(<T as Config>::WeightInfo::add_authorization_full::<T::AccountId>(&data))]
+        #[pallet::call_index(10)]
         pub fn add_authorization(
-            origin,
+            origin: OriginFor<T>,
             target: Signatory<T::AccountId>,
             data: AuthorizationData<T::AccountId>,
-            expiry: Option<T::Moment>
-        ) {
+            expiry: Option<T::Moment>,
+        ) -> DispatchResult {
             Self::base_add_authorization(origin, target, data, expiry)?;
+            Ok(())
         }
 
         /// Removes an authorization.
-        /// _auth_issuer_pays determines whether the issuer of the authorisation pays the transaction fee
-        #[weight = <T as Config>::WeightInfo::remove_authorization()]
+        /// `_auth_issuer_pays` determines whether the issuer of the authorisation pays the transaction fee
+        #[pallet::weight(<T as Config>::WeightInfo::remove_authorization())]
+        #[pallet::call_index(11)]
         pub fn remove_authorization(
-            origin,
+            origin: OriginFor<T>,
             target: Signatory<T::AccountId>,
             auth_id: u64,
             _auth_issuer_pays: bool,
-        ) {
```

Comment:
The API check fails do to the parameter `_auth_issuer_pays` being renamed to `auth_issuer_pays`.  It seems the new FRAME v2 codegen drops the `_` prefix.